src/meta/src/lib.rs

```diff
@@ -48,26 +48,19 @@ mod stream;
 
 use std::time::Duration;
 
-use clap::{ArgEnum, Parser};
+use clap::Parser;
 pub use error::{MetaError, MetaResult};
 
 use crate::manager::MetaOpts;
 use crate::rpc::server::{rpc_serve, AddressInfo, MetaStoreBackend};
 
-#[derive(Copy, Clone, Debug, ArgEnum)]
-pub enum Backend {
-    Mem,
-    Etcd,
-}
-
+/// CLI arguments received by meta node. Overwrites fields in
+/// [`risingwave_common::config::MetaConfig`].
 #[derive(Debug, Clone, Parser)]
 pub struct MetaNodeOpts {
-    // TODO: rename to listen_address and separate out the port.
-    #[clap(long, default_value = "127.0.0.1:5690")]
-    listen_addr: String,
+    #[clap(long)]
+    listen_addr: Option<String>,
 
-    /// The endpoint for this meta node, which also serves as its unique identifier in cluster
-    /// membership and leader election.
```

Comment:
Why delete these rustdoc comments? 👀 AFAIK `clap` also uses it to generate the text for `--help`, so please make sure all of the arguments are well rustdoced.

Reply:
I didn't realize.